lib/views/activites/picture_select_activity_view.dart

```diff
@@ -62,7 +65,7 @@ class _PictureSelectActivityViewState extends State<PictureSelectActivityView> {
                     ? BodyText(
                         "You Answered:",
                       )
-                    : null,
+                    : Container(),
```

Comment:
why did you change these back to empty containers

Reply:
Children of columns/rows cannot be null